src/Microsoft.VisualStudio.Threading.Analyzers.Tests/VSTHRD003UseJtfRunAsyncAnalyzerTests.cs

```diff
@@ -1080,6 +1080,41 @@ public Task GetTask()
             await Verify.VerifyAnalyzerAsync(test, expected);
         }
 
+        [Fact(Skip = "Test is not yet implemented since it needs TWO source files.")]
```

Comment:
We should address the TODO in this test and then stop skipping it. @sharwell can you advise how to supply *two* source files to an analyzer test? Your roslyn analyzer test framework doesn't seem to support that.

Reply:
➡️ Now resolved in #422